crates/turborepo-lib/src/shim.rs

```diff
@@ -80,7 +87,7 @@ impl ShimArgs {
             Err(anyhow!("No value assigned to `--cwd` argument"))
         } else {
             let cwd = if let Some(cwd) = cwd {
-                cwd
+                cwd.canonicalize()?
```

Comment:
Should this be `fs_canonicalize(cwd)?` so we get the dunce version when on windows?